src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -38,6 +38,8 @@ public class JedisClusterInfoCache {
   private final Map<String, ConnectionPool> nodes = new HashMap<>();
   private final ConnectionPool[] slots = new ConnectionPool[Protocol.CLUSTER_HASHSLOTS];
   private final HostAndPort[] slotNodes = new HostAndPort[Protocol.CLUSTER_HASHSLOTS];
+  private List<ConnectionPool>[] replicaSlots;
+  private List<HostAndPort>[] replicaSlotNodes;
```

Comment:
Please check. There may not any use left for `replicaSlotNodes`.

Reply:
Yes, the `replicaSlotNodes` is not used and I have remove it.